ddtrace/ext/aws.py

```diff
@@ -17,7 +17,8 @@ def unpacking_args(args, args_name, traced_args_list):
         traced_args_list: list of names of the args we want to trace
     Returns a list of (arg name, arg) of the args we want to trace
     The number of args being variable from one call to another, this function
-    will parse t"""
+    will parse t
```

Comment:
(a remnant from before)

```suggestion
    will parse it
```
?